flink-connectors/flink-connector-elasticsearch-base/src/main/java/org/apache/flink/streaming/connectors/elasticsearch/table/ElasticsearchOptions.java

```diff
@@ -29,9 +30,8 @@
 
 import static org.apache.flink.configuration.description.TextElement.text;
 
-/**
- * Options for {@link org.apache.flink.table.factories.DynamicTableSinkFactory} for Elasticsearch.
- */
+/** Options for the Elasticsearch connector. */
+@PublicEvolving
 public class ElasticsearchOptions {
     /**
```

Comment:
nit: how about we move helper classes all the way down and split each option definition by an empty line for better readability.